### DISCUSSION TITLE
bot lags

### DISCUSSION THREAD

**Discussion by ITEXNIKJS** [PrismarineJS/mineflayer#2721] [Q&A]
The bot starts lagging when there is a certain time in one direction, as soon as the direction changes, the lags disappear. Could these be server limitations?

**Comment by amoraschi** [comment]
What's your code?

**Reply by ITEXNIKJS** [reply]
![image](https://user-images.githubusercontent.com/94114335/182020700-cb472c1e-9e5d-4192-8c01-c39bed3e1b29.png)

**Reply by amoraschi** [reply]
physicsTick is called every 50ms, are you sure you want to be using that event?

**Reply by ITEXNIKJS** [reply]
Thats wrong with it?

**Reply by amoraschi** [reply]
Sorry I forgot, I wanted to say that it could be doing something that's resource expensive
